hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/ITTestDataStreamWrite.java

```diff
@@ -186,11 +182,12 @@ public void testWriteCopyOnWriteWithClustering(boolean sortClusteringEnabled) th
     writeWithClusterAndCheckExpected(conf, "cow_write_with_cluster", 1, EXPECTED);
   }
 
-  @Disabled("HUDI-9196")
   @ParameterizedTest
   @ValueSource(strings = {"COPY_ON_WRITE", "MERGE_ON_READ"})
   public void testStreamWriteWithIndexBootstrap(String tableType) throws Exception {
     Configuration conf = TestConfigurations.getDefaultConf(tempFile.toURI().toString());
+    conf.set(FlinkOptions.COMPACTION_ASYNC_ENABLED, false);
+    conf.set(FlinkOptions.COMPACTION_DELTA_COMMITS, 1);
```

Comment:
Flink does not have inline compaction.

Reply:
Writing pipeline in this test is customized, not same with that in `HoodieTableSink`. Compact operator is added if the table type is MOR, see Line 285.